### ISSUE TITLE
Installation problems

### ISSUE THREAD

**Issue by gridge** [mcviz/mcviz#88]
Hi,
 I was trying to install/run this from the git repository.
Once I try to run python mcviz_bootstrap.py -v (to get a bit more messages) I obtain what is pasted below.
# 

Directory env/lib/python2.7 already exists
Symlinking Python bootstrap modules
  Ignoring built-in bootstrap module: posix
  Cannot import bootstrap module: nt
  Ignoring built-in bootstrap module: zlib
Directory env/lib/python2.7/site-packages already exists
Content env/lib/python2.7/site.py already in place
Content env/lib/python2.7/orig-prefix.txt already in place
Content env/lib/python2.7/no-global-site-packages.txt already in place
Directory env/bin already exists
New python executable in env/bin/python
Changed mode of env/bin/python to 0775
Testing executable with env/bin/python -c "
import sys
prefix = sys.prefix
if sys.version_info[0] == 3:
    prefix = prefix.encode('utf8')
if hasattr(sys.stdout, 'detach'):
    sys.stdout = sys.stdout.detach()
elif hasattr(sys.stdout, 'buffer'):
    sys.stdout = sys.stdout.buffer
sys.stdout.write(prefix)
"
Got sys.prefix result: u'/opt/mcviz/env'
Directory env/lib/python2.7/distutils already exists
Content env/lib/python2.7/distutils/**init**.py already in place
Content env/lib/python2.7/distutils/distutils.cfg already in place
Using existing distribute egg: ./distribute-0.6.24.tar.gz
Installing distribute..............................................................................................................................................................................................done.
Installing existing pip-1.0.2.tar.gz distribution: dist/pip-1.0.2.tar.gz
Installing pip...
  Processing pip-1.0.2.tar.gz
  Running pip-1.0.2/setup.py -q bdist_egg --dist-dir /tmp/easy_install-cMdk9F/pip-1.0.2/egg-dist-tmp-hLRQcV
  warning: no files found matching '_.html' under directory 'docs'
  warning: no previously-included files matching '_.txt' found under directory 'docs/_build'
  no previously-included directories found matching 'docs/_build/_sources'
  pip 1.0.2 is already the active version in easy-install.pth
  Processing dependencies for pip==1.0.2
  Finished processing dependencies for pip==1.0.2
...Installing pip...done.
Submodule 'mcviz.examples' () registered for path 'mcviz.examples'
Submodule 'mcviz.jet' () registered for path 'mcviz.jet'
Obtaining file:///opt/mcviz
  Running setup.py egg_info for package from file:///opt/mcviz
    The required version of distribute (>=0.6.28) is not available,
    and can't be installed while this script is running. Please
    install a more recent version first, using
    'easy_install -U distribute'.

```
(Currently using distribute 0.6.24 (/opt/mcviz/env/lib/python2.7/site-packages/distribute-0.6.24-py2.7.egg))
Complete output from command python setup.py egg_info:
The required version of distribute (>=0.6.28) is not available,
```

and can't be installed while this script is running. Please

install a more recent version first, using

'easy_install -U distribute'.

(Currently using distribute 0.6.24 (/opt/mcviz/env/lib/python2.7/site-packages/distribute-0.6.24-py2.7.egg))

---

Command python setup.py egg_info failed with error code 2
Storing complete log in /home/spagan/.pip/pip.log
Content env/bin/activate already in place
Content env/bin/activate.fish already in place
Content env/bin/activate_this.py already in place
Content env/bin/activate.csh already in place
# 

I'm using python 2.7.3
In my sistem distribute 0.6.35 is installed but I did not find a way to have this to use it.

Any help is very appreciated. Thanks and looking forward to play with your nice utility (yes, I got bored as well to check events from a list...)

Simone
